task/fbc-target-index-pruning-check/0.1/fbc-target-index-pruning-check.yaml

```diff
@@ -128,6 +129,8 @@ spec:
 
         failure_num=0
         TESTPASSED=true
+        # Allowed pruned channels are those that start with dev-preview, pre-ga, or candidate (regardless of what follows)
+        fbc_allowed_pruned_channels="^(dev-preview|pre-ga|candidate).*"
```

Comment:
It is also possible for a channel to be named `some-operator-candidate`

Reply:
I think a more generic approach like `.*(dev-preview|pre-ga|candidate).*` would suit better. thoughts?